### ISSUE TITLE
Blind Trust Before Verification

### ISSUE THREAD

**Issue by gerg5c42g542g2c54g52c** [ChatSecure/ChatSecure-iOS#1088]
Info: https://gultsch.de/trust.html

Please prioritize implementing this as this is one of the biggest problem with using ChatSecure with normal people. Right now, I have to send a message from my other device that they should go to settings and enable that new fingerprint and they go there and there's a big mess, lots of fingerprints (old, unused etc.) and enable it. Otherwise, they don't even know that I tried telling them something as they can't read my messages. It irritates them and they hate the app (they only hate bugs other than this) and it irritates me as I have to go through this every time.

**Comment by sindastra** [comment]
I agree. The way Conversations (Android) does it, is much more user friendly and a good example of how it could be done. Perhaps make blind trust the default but add an advanced option to disable blind trust, for advanced users that want to manage keys themselves. That way it should be well balanced and user friendly in my opinion.

**Comment by heldzz** [comment]
I agree, too. Even more this would be good as Chatsecure does not inform the user about a non-decryptable message - it just ignores it.
